README.md

```diff
@@ -42,15 +44,28 @@ sudo docker run -d \
   factoriotools/factorio
 ```
 
+#### macOS
+
+```shell
+mkdir -p ~/.factorio
+docker run -d \
+  -p 34197:34197/udp \
+  -p 27015:27015/tcp \
+  -v ~/.factorio:/factorio \
+  --name factorio \
+  --restart=always \
+  factoriotools/factorio
+```
+
 For those new to Docker, here is an explanation of the options:
 
 * `-d` - Run as a daemon ("detached").
 * `-p` - Expose ports.
-* `-v` - Mount `/opt/factorio` on the local file system to `/factorio` in the container.
+* `-v` - Mount `/opt/factorio` (or `~/.factorio` for macOS) on the local file system to `/factorio` in the container.
```

Comment:
I would rather add some note somewhere that macOS user should replace ``/opt/factorio`` with ``~/.factorio`` because we have mentioned the first path multiple times at other places.